src/python/pants/notes/2.2.x.md

```diff
@@ -9,6 +9,64 @@ https://raw.githubusercontent.com/pantsbuild/setup/2f079cbe4fc6a1d9d87decba51f19
 This release also removes the ability to use `%(pants_bootstrapdir)s` and `%(pants_configdir)s` in
 interpolations in your config file.
 
+## 2.2.0rc0 (Dec 28, 2020)
+
+### New Features
+
+* Infer imports of the style __import__("string_literal"). ([#11326](https://github.com/pantsbuild/pants/pull/11326))
+
+* Make the number of threads used to run @rules in the Engine configurable. ([#11325](https://github.com/pantsbuild/pants/pull/11325))
+
+* Expose --resolver-version to support using the new pip resolver, deprecate --cache-ttl. ([#11349](https://github.com/pantsbuild/pants/pull/11349))
+
+### User API Changes
+
+* Allow newer versions of pytest (6.2.x) ([#11381](https://github.com/pantsbuild/pants/pull/11381))
+
+* Upgrade packaging from 20.7 to 20.8 ([#11343](https://github.com/pantsbuild/pants/pull/11343))
+
+* Deprecate the `sources` field for `pex_binary` and `python_awslambda` ([#11332](https://github.com/pantsbuild/pants/pull/11332))
+
+### Plugin API Changes
+
+* Deprecate Subsystems as global singletons ([#11388](https://github.com/pantsbuild/pants/pull/11388))
+
+* Add type annotations around Subsystem class ([#11389](https://github.com/pantsbuild/pants/pull/11389))
```

Comment:
I think internal only. Notably, we still don't follow PEP 561 and MyPy doesn't understand Pants when it's used as a library.